go/iot/arduino/protocol.txt

```diff
@@ -36,23 +36,28 @@ When the program starts, the following view would be registered.
 
 Before the registration, the Raspberry Pi would not make any response to the the request from Arduino.
 
-After the registration, the Raspberry Pi would make a negative or positive response to any record request from Arduino.
-
-When the Pi would make a negative response:
-    1. The measure doesn't bind to any views.
-    2. The tag key doesn't bind to any views.
-    2. Pi could not parse the encapsulated data because of the format error.
-    3. The number of tag keys is not equal to the number of tag values.
-
-When the Pi successfully parses and records data, it would make a positive response to the request. Note that it would
-still responds positively if there is something wrong with the OpenCensus framework such as networking failure.
+After the registration, the Raspberry Pi would make a response to the record request from Arduino.
 
 Typical example for the response by Raspberry Pi would be as below:
 {
 	"Code": 200 or 404
 	"Info": "Record Successfully!" or "Registration Not Finished" or ...
 }
 
+In the following cases Pi would make response with the Code 404:
+    1. The measure doesn't bind to any views.
+    2. Pi could not parse the encapsulated data because of the format error.
+    3. The number of tag keys is not equal to the number of tag values.
```

Comment:
This case is no longer applicable after representing tags as a map.

Reply:
gocha